src/main/java/uk/gov/hmcts/reform/dg/docassembly/dto/PDFConversionDto.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.dg.docassembly.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.validation.constraints.NotNull;
+import java.util.UUID;
+
+@Setter
+@Getter
+@ToString
+public class PDFConversionDto {
```

Comment:
We don't a Dto to hold just one attribute. Instead we can get the documentId as path variable.